taskcluster/ci/signing/kind.yml

```diff
@@ -12,22 +12,27 @@ transforms:
 kind-dependencies:
     - build
 
+
 job-template:
     description: Sign Fenix
     worker-type:
         by-build-type:
-            (fennec-production|nightly|beta|production|android-test-nightly):
+            (fennec-.+|nightly|beta|production|android-test-nightly):
                 by-level:
                     '3': signing
                     default: dep-signing
             default: dep-signing
     worker:
         signing-type:
             by-build-type:
-                fennec-production:
+                fennec-(beta|production):
```

Comment:
I see you've added beta as well. In https://github.com/mozilla-releng/scriptworker-scripts/pull/94 you only added credentials for `nightly`. If you're exposing the Fennec beta too, shouldn't we enrich https://github.com/mozilla-releng/scriptworker-scripts/pull/94 to contain the Fennec beta credentials too?

Reply:
The Fennec beta creds are the same as the fennec prod ones. Because of historical reasons Fennec Nightly and Fennec Aurora shared the same key (the "nightly" key), and same for Fennec Beta and Fennec Release (the release one, called "production" here).